robot_skills/src/robot_skills/arm/grasp_detector.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, robot_name, tf_buffer, wrench_topic):
         self.latest_msg = None
         self.msg_list = []
         self.threshold_torque_y = -0.45
-        self.measuring_for = 15
+        self.measuring_for = 15.to_sec()
```

Comment:
This should be a rospy Duration
```suggestion
        self.measuring_for = rospy.Duration(15)
```